phpunit/class-wp-theme-json-resolver-test.php

```diff
@@ -1350,6 +1393,62 @@ public function test_get_resolved_theme_uris() {
 		$this->assertSame( $expected_data, $actual );
 	}
 
+	public function test_get_resolved_theme_uris_in_variations() {
+		register_block_style(
+			'core/group',
+			array(
+				'name'       => 'group-background-twinky',
+				'label'      => 'Group block twinky background',
+				'style_data' => array(
```

Comment:
We probably need to expand this test block style a little so it covers element styles at both the variation root and inner block levels.